api/testdata/complete-scenarios/test6.json

```diff
@@ -1731,7 +1730,6 @@
                       "first": "",
                       "firstInitialOnly": false,
                       "middle": "",
-                      "hideMiddleName": true,
```

Comment:
Same here, not sure if these values are no longer needed, or were just accidentally removed.

Reply:
Those changes weren't manual – I loaded up test case 6, made my changes to spouse, and used `form` to export it back out. I had assumed that those were things that had changed in the code base. Should they still be there? It is what eApp saves right now.

Reply:
It looks like what is happening is the prop `hideMiddleName` is being set on certain `<Name>` components, which visually accomplishes the goal, but it does not appear the value is being stored in the database, which it should, since it is used in validation. But that's all unrelated to this change, so we could just restore these values and I can open up another issue, or we could remove them, but it may cause a "phantom" error on the frontend.

Reply:
Actually, looks like I already wrote this up: https://github.com/18F/e-QIP-prototype/issues/1293. Rest of the changes look good though, so I'll give it a 👍